packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -567,6 +568,16 @@ export class FocusZone extends React.Component<IFocusZoneProps> implements IFocu
             break;
           }
           return;
+        case KeyCodes.pageDown:
+          if (direction !== FocusZoneDirection.horizontal && this._movePageDown()) {
```

Comment:
>(direction !== FocusZoneDirection.horizontal [](start = 13, length = 44)

Do we not want pageUp/PageDown to have the behavior you're proposing when we are in a horizontal FocusZone? Having pageUp/PageDown work on horizontal FocusZones seems like a valid thing that a user may want to do.

Reply:
yes , you are right.
i think we need it in the all cases(horizontal ,vertical)

Reply:
It's not clear to me if there's an action needed here. Does the PR need to be updated?

Reply:
yes, i update it now jason :)